src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -175,45 +175,59 @@ some/path/MyApp/
     │   ├── App.framework
     │   ├── FlutterPluginRegistrant.framework
     │   └── example_plugin.framework (each plugin with iOS platform code is a separate framework)
-      ├── Profile/
-      │   ├── Flutter.framework
-      │   ├── App.framework
-      │   ├── FlutterPluginRegistrant.framework
-      │   └── example_plugin.framework
-      └── Release/
-          ├── Flutter.framework
-          ├── App.framework
-          ├── FlutterPluginRegistrant.framework
-          └── example_plugin.framework
+    ├── Profile/
+    │   ├── Flutter.framework
+    │   ├── App.framework
+    │   ├── FlutterPluginRegistrant.framework
+    │   └── example_plugin.framework
+    └── Release/
+        ├── Flutter.framework
+        ├── App.framework
+        ├── FlutterPluginRegistrant.framework
+        └── example_plugin.framework
 ```
 
 {{site.alert.tip}}
-With Xcode 11 installed, you can generate [XCFrameworks][] instead of universal frameworks by adding
-the flags `--xcframework --no-universal`.
+  With Xcode 11 installed, you can generate [XCFrameworks][] instead of universal frameworks by adding
+  the flags `--xcframework --no-universal`.
 {{site.alert.end}}
 
-Embed the generated frameworks into your existing application in Xcode. For example, you can
-drag the frameworks from `some/path/MyApp/Flutter/Release/` in Finder
-into your targets's build settings > General > Frameworks, Libraries, and Embedded Content. Then, select
-"Embed & Sign" from the drop-down list.
+Embed and link the generated frameworks into your existing application in Xcode.
+There are multiple ways to do this—use the method that is best for your project.
```

Comment:
nit: `this - use`?

Reply:
I don't usually put spaces around em dashes, though it looks like newspapers sometimes do this: https://www.thepunctuationguide.com/em-dash.html#spaces
It doesn't have spaces around it in other parts of the website, I'll keep it as-is for consistency unless @sfshaza2 wants me to change this: https://flutter.dev/docs/get-started/flutter-for/react-native-devs#entry-point

Reply:
You are correct, @jmagman!  From our style guide:  https://developers.google.com/style/dashes